src/formatDate.js

```diff
@@ -8,7 +8,66 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldYear = yearFormatAndPlace(fromFormat);
```

Comment:
The function `yearFormatAndPlace` is being called before its declaration. While JavaScript hoisting allows function declarations to be lifted to the top, it's generally a good practice to call functions after they have been declared to improve code readability and maintainability.